File: app/src/test/java/com/example/androidfundamentals/domains/AddressTest.java
package com.example.androidfundamentals.domains;

import org.junit.Test;

import static org.junit.Assert.*;

public class AddressTest {

    @Test
    public void shouldReturnAddressText() {
        Address address = new Address("31030030","Rua Córrego da Mata", "de 213/214 ao fim", "Horto", "Belo Horizonte", "MG");

        String addressAsText = address.toText();

        assertEquals("Rua Córrego da Mata, de 213/214 ao fim - Horto, Belo Horizonte - MG", addressAsText);
    }
}

File: app/src/test/java/com/example/androidfundamentals/domains/AddressTest.kt
package com.example.androidfundamentals.domains

import org.junit.Assert.assertEquals
import org.junit.Test

class AddressTest {

    @Test
    fun shouldReturnAddressText() {
        val address = Address("31030030", "Rua Córrego da Mata", "de 213/214 ao fim", "Horto", "Belo Horizonte", "MG")

        val addressAsText = address.toText()

        assertEquals("Rua Córrego da Mata, de 213/214 ao fim - Horto, Belo Horizonte - MG", addressAsText)
    }
}